aries_vcx/src/handlers/out_of_band/receiver.rs

```diff
@@ -1,25 +1,44 @@
 use std::clone::Clone;
 
+
+use indy_sys::WalletHandle;
+
```

Comment:
Extra line. How about we run rustfmt and clippy in CI? Rustfmt can (among other things) sort imports which is quite nice.

Reply:
fixed and created [issue](https://github.com/hyperledger/aries-vcx/issues/540)